promql/quantile.go

```diff
@@ -95,15 +95,15 @@ type metricWithBuckets struct {
 // and another bool to indicate if small differences between buckets (that
 // are likely artifacts of floating point precision issues) have been
 // ignored.
-func bucketQuantile(q float64, buckets buckets) (float64, bool, bool) {
+func bucketQuantile(q float64, buckets buckets) (float64, float64, float64, float64, bool, bool) {
```

Comment:
Also I'd named the return values for clarity in the signature, probably makes the code little but simpler as well